pkg/server/restart.go

```diff
@@ -342,6 +350,10 @@ func loadImages(ctx context.Context, cImages []containerd.Image, provider conten
 		// imgs len must be > 0, or else the entry will not be created in
 		// previous loop.
 		i := imgs[0]
+		if ok, _, _, _, err := imagemanage.Check(ctx, provider, i.Target(), platforms.Default()); !ok || err != nil {
+			glog.Warningf("Failed to check image readiness for %q: %v", i.Name(), err)
```

Comment:
Let's log different for `!ok` and `err != nil`.

Reply:
Failed to check image content readiness for %q: %v

Reply:
done